starport/pkg/spn/chain.go

```diff
@@ -28,7 +28,7 @@ func PaginateChainListing(key []byte, limit uint64) ChainListOption {
 }
 
 // ChainList lists chain summaries
-func (c *Client) ChainList(ctx context.Context, accountName string, options ...ChainListOption) (chains []Chain, nextPageKey []byte, err error) {
+func (c *Client) ChainList(ctx context.Context, accountName string, prefix string, options ...ChainListOption) (chains []Chain, nextPageKey []byte, err error) {
```

Comment:
Can we pass this `prefix` as an option (of type `ChainListOption`) to this func?

```go
func SearchPrefixChainListing(prefix string) ChainListOption { ... }

Reply:
Thanks for the suggestion, I thought it was specific to pagination